util/daemonize.c

```diff
@@ -53,29 +88,19 @@ daemonize(const char * spid)
 			warnp("close");
 			goto err1;
 		}
-		do {
-			switch (read(fd[0], &dummy, 1)) {
```

Comment:
I think we can get rid of the dummy variable now?

Reply:
Unfortunately it's needed for writing to `fd`:

```
        /* Tell the parent to suicide. */
        if (noeintr_write(fd[1], &dummy, 1) == -1) {
                warnp("write");
                goto die;       
        }
```

I was very tempted to move that `noeintr_write()` into a separate function (maybe called `sendbyte()`?), to get rid of `dummy` and to be semantically clearer, but I figured that would be over-engineering it.

Reply:
Right, I forgot about that.  Yeah no point moving that code.